pkg/misconf/scanner.go

```diff
@@ -493,8 +493,10 @@ func ResultsToMisconf(configType types.ConfigType, scannerName string, results s
 		query := fmt.Sprintf("data.%s.%s", result.RegoNamespace(), result.RegoRule())
 
 		ruleID := result.Rule().AVDID
-		if result.RegoNamespace() != "" && len(result.Rule().Aliases) > 0 {
-			ruleID = result.Rule().Aliases[0]
+		if ruleID == "" {
+			if result.RegoNamespace() != "" && len(result.Rule().Aliases) > 0 {
```

Comment:
nit:
it looks like we don't need nested `if`:
```suggestion
		if ruleID == "" && result.RegoNamespace() != "" && len(result.Rule().Aliases) > 0 {
```